com.unity.multiplayer.mlapi/Runtime/NetworkVariable/INetworkVariable.cs

```diff
@@ -8,6 +8,12 @@ namespace MLAPI.NetworkVariable
     /// </summary>
     public interface INetworkVariable
     {
+        /// <summary>
+        /// Gets or sets the name of the network variable's instance
+        /// (MemberInfo) where it was declared.
+        /// </summary>
+        string Name { get; set; }
```

Comment:
yeah, I think this could be just `string Name { get; }` and concrete interface implementations could do `public string Name { get; internal set; }` just fine.